common/src/test/java/com/google/bos/udmi/monitoring/LogTailTest.java

```diff
@@ -0,0 +1,116 @@
+package com.google.bos.udmi.monitoring;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+import com.google.cloud.MonitoredResource;
+import com.google.cloud.logging.LogEntry;
+import com.google.cloud.logging.LogEntryServerStream;
+import com.google.cloud.logging.Severity;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.jupiter.api.Test;
+import org.mockito.MockitoAnnotations;
+
+
+public class LogTailTest {
+
+  final static String TEST_PROJECT_NAME = "XXXtestXXXCantBeARealProject";
+  final static String[] TEST_MAIN_ARGS = {"-p", TEST_PROJECT_NAME};
+
+
+  private AutoCloseable closeable;
+
+  private LogTail getLogTailMock() {
+    LogTail logTail = new LogTail(TEST_PROJECT_NAME);
+    return (LogTail) spy(logTail);
+  }
+
+  @Before
+  public void openMocks() {
+    closeable = MockitoAnnotations.openMocks(this);
+  }
+
+  @After
+  public void releaseMocks() throws Exception {
+    closeable.close();
```

Comment:
what's the purpose/impact of this closeable.close()?  Asking because I don't have anything similar in the code I've done so I don't know if I'm missing something or not...